package.json

```diff
@@ -50,7 +50,7 @@
     "react-dnd-touch-backend": "^16.0.1",
     "react-dom": "^18",
     "react-redux": "^9.1.0",
-    "tapwrite": "^1.0.31",
+    "tapwrite": "^1.0.37",
```

Comment:
This version doesn't include `editorClass` IMO. Could you please add the latest here since it includes an important prop and changes?